C7/PlayerCamera.cs

```diff
@@ -44,4 +44,13 @@ public partial class PlayerCamera : Camera2D
 		Vector2 target = map.tileToLocal(tile);
 		Position = target;
 	}
+
+	public bool isTileInView(Tile tile, MapView map) {
+		Rect2 visible = getVisibleWorld();
+		Vector2 target = map.tileToLocal(tile);
+		float size = 30;
```

Comment:
does this account for zoom level? is there a constant that can be referenced instead of the literal 30?

Reply:
It does, the 30 is 30 pixels in world space. The Rect2 returned from getVisibleWorld accounts for zoom since it uses the camera's transform from GetCanvasTransform. I can make 30 a constant in the camera class. Really, it should check if any of the orthogonal tile is in view, but a conservative bounding box inside the tile is much easier to calculate and I assume a fast bounds check